src/ContainerDetails.jsx

```diff
@@ -43,6 +44,17 @@ const ContainerDetails = ({ container }) => {
                         <DescriptionListDescription>{utils.quote_cmdline(container.Config.Cmd)}</DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    {Boolean(container.Config?.Labels?.PODMAN_SYSTEMD_UNIT) && (container.uid === 0 || container.uid === null) &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm>{_("Systemd service")}</DescriptionListTerm>
+                        <DescriptionListDescription>
+                            <Button variant="link" isInline onClick={
+                                () => cockpit.jump(`/system/services#/${container.Config?.Labels?.PODMAN_SYSTEMD_UNIT}` + (container.uid === null ? "?owner=user" : ""))}>
```

Comment:
This was my main fear about the podman multi-user request: it'd soon percolate through all the other pages, like logs, services, terminal, etc. :thinking:

Reply:
Yes, realized that as well and that is why it is only visible for the "admin session" and "superuser".